pkg/network/tracer/tracer_usm_linux_test.go

```diff
@@ -59,12 +59,16 @@ func classificationSupported(config *config.Config) bool {
 	return kprobe.ClassificationSupported(config)
 }
 
+func usmSupportedBuildModes(t *testing.T) []ebpftest.BuildMode {
+	return supportedBuildModes(t)
```

Comment:
that might contain fentry, which is unsupported for USM

Reply:
Good catch; fixed